apps/todoapp/src/app/todos/components/user-info/user-info.component.scss

```diff
@@ -1,8 +1,40 @@
 .user {
-  display: flex;
-  align-items: center;
+  display: table-row;
+}
+
+.user * {
+  display: table-cell;
+  padding: 1px 0;
 }
 
 .user p {
-  margin-right: 8px;
+  padding-left: 10px;
```

Comment:
make a padding on the parent component instead of padding left on the first child and padding right on the last child

Reply:
this is exactly what i wanted to achieve not having padding on parent element only a little bit of padding for both child elements to the left/right